edb/server/protocol/auth_ext/ui.py

```diff
@@ -170,7 +166,7 @@ def render_signup_page(
 
       {_render_error_message(error_message)}
 
-      <input type="hidden" name="provider" value="{provider_id}" />
+      <input type="hidden" name="provider" value="{provider_name}" />
```

Comment:
Change the below email input from `<input type="text">` to `<input type="email">` (ditto for sign in if not there already)